### PR TITLE
Add DCO prompt in PR template

### PR DESCRIPTION
**Pull request by WangTaoTheTonic**
Signed-off-by: WangTaoTheTonic <dooku.wt@antfin.com>

## Why are these changes needed?
We require DCO in the PR, but people always forget it until the PR being created.
Putting a prompt line in the PR template may help.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 ## Checks
 
+- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
```
Comment (jovany-wang) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: every commit or the latest commit?

Comment (WangTaoTheTonic) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: Everty commit should be signed off, see:
image
I also remember once I only signed off the last one but the DCO failed.

Comment (WangTaoTheTonic) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: And a simple test is here: https://github.com/ray-project/ray/pull/27082

Comment (jovany-wang) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: GOT IT!

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**